packages/components/src/experience-control/style.scss

```diff
@@ -27,60 +27,65 @@ $experience-types: (
 );
 
 .experience-control {
-	display: flex;
-	flex-direction: column;
-	gap: 12px;
+	.components-base-control__label {
+		@include heading-medium;
+		margin-block-end: 12px;
+		text-transform: uppercase;
+		color: var(--color-text);
+	}
```

Comment:
We don't want to override the Core label styles, these are standardized across components. Is it possible to update the A4A screen so it uses standard Core components/styling?